RealEstateDataCleaning.py

```diff
@@ -8,113 +8,160 @@
 import pandas as pd
 import numpy as np
 
+# aea: argparse here
 input_file = 'Real Estate Model - Candler Park.csv'
```

Comment:
The instinct to put `input_file` and `output_file` into variables is a good one.  The next step up here is to use `argparse` to pass those variables on the command line.  I'd personally refactor this to something like:

```
if __name__ == '__main__':
    import argparse
    parser = argparse.ArgumentParser(usage="Process house data")
    parser.add_argument("-i", dest="input_file",
                        help="input csv file", type=str, 
                        default="Real Estate Model - Candler Park.csv")
    args = parser.parse_args()
    data_frame = pd.read_csv(args.input_file)
```